Directory.Packages.props

```diff
@@ -127,7 +127,7 @@
     <PackageVersion Include="MySqlConnector.Logging.Microsoft.Extensions.Logging" Version="2.1.0" />
     <PackageVersion Include="NATS.Net" Version="2.4.0" />
     <PackageVersion Include="Npgsql.DependencyInjection" Version="8.0.4" />
-    <PackageVersion Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="8.0.8" />
+    <PackageVersion Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="$(NpgsqlEntityFrameworkCorePostgreSQLPackageVersion)" />
```

Comment:
(low pri question): How should we think about what versions we put in Directory.Packages.props and which we put in Versions.props? I guess I thought the ones in Versions.props were updated by darc, but I don't imagine this one will be.

Reply:
Good question, this variable is in `versions.props` because this is were the TFM choice is made, but not sure how darc will handle the split even for other packages it can update.